NuCivic/dkan
# Fix featured groups view access CIVIC-5706

Issue: CIVIC-5706
**1.13 PR**
## Description

The Featured Groups view isn't visible to anonymous users.

## How to reproduce

1. add the `Featured Group grid pane` view to a page
2. view the page as an anonymous user

## QA Steps

1. log in as an administrator
2. click 'Content' then 'edit' to edit the welcome page
3. use the 'customize display' button, click the content link, add the `Featured Group grid pane` view to the page
4. view the page as an anonymous user
5. confirm you can see the group teasers
